### PR TITLE
add workflow

### PR DESCRIPTION
**Pull request by IOVgomezdn**
- separate testnet and mainnet environment files
- add new workflow for testnet environment
- add env-cmd library to select which .env to use

### CODE REVIEW

**Review by lucasvuotto — COMMENTED** [review]
Looks alright. I'd take the chance and already fill `.env.mainnet`.

This is missing `package-lock.json` changes; is that intended?

File: .github/workflows/testnet-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.TESTNET_ROLE_ARN }}
+          aws-region: ${{ secrets.TESTNET_AWS_REGION }}
+
+      - name: Deploy site to S3
+        run: |
+          aws s3 sync --delete --only-show-errors dist/ ${{ secrets.TESTNET_S3_BUCKET }}
```
Comment (lucasvuotto) [inline .github/workflows/testnet-deploy.yml:37]: This is missing KMS-related options:

```suggestion
          aws s3 sync --sse aws:kms --sse-kms-key-id ${{ secret.TESTNET_KMS_KEY_ID }} --delete --only-show-errors dist/ ${{ secrets.TESTNET_S3_BUCKET }}
```

**Review by lucasvuotto — CHANGES_REQUESTED** [review]

File: src/config/menu.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const items = [
   'blocks',
   'transactions',
   'addresses',
-  'tokens',
+  'tok3ns',
```
Comment (jonathansmirnoff) [inline src/config/menu.js:21]: is this a typo?

File: .env.mainnet [hunk 3]
```diff
@@ -1,6 +1,6 @@
 VUE_APP_WS_URL=wss://backend.explorer.rsk.co
+VUE_APP_APPS_URL=""
 VUE_APP_STATS_URL=https://stats.rsk.co
-VUE_APP_GA_TAG=''
-VUE_APP_HOTJAR_ID=''
+VUE_APP_GA_TAG=GTM-56J7JTD
+VUE_APP_HOTJAR_ID=1857874
```
Comment (jonathansmirnoff) [inline .env.mainnet:5]: Is OK to push these values?

**Review by jonathansmirnoff — COMMENTED** [review]
I found small things to double-check that we are OK

File: .env.mainnet (hunk 3 above)
Comment (lucasvuotto) [reply .env.mainnet:5]: Yes, they're loaded by the client on page visit.

File: src/config/menu.js (hunk 2 above)
Comment (lucasvuotto) [reply src/config/menu.js:21]: No, the commit introducing it makes its usage clear: it's to test the workflow. https://github.com/rsksmart/rsk-explorer/pull/189/commits/c1052cb02e98e34bf78e3f529c44474a57c65ef0 .

**Review by jonathansmirnoff — APPROVED** [review]

File: .github/workflows/testnet-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.TESTNET_ROLE_ARN }}
+          aws-region: ${{ secrets.TESTNET_AWS_REGION }}
+
+      - name: Deploy site to S3
+        run: |
+          aws s3 sync --sse aws:kms --sse-kms-key-id ${{ secret.TESTNET_KMS_KEY_ID }} --delete --only-show-errors dist/ ${{ secrets.TESTNET_S3_BUCKET }}
```
Comment (jonathansmirnoff) [inline .github/workflows/testnet-deploy.yml:37]: aws s3 sync --sse aws:kms --sse-kms-key-id ${{ secrets.TESTNET_KMS_KEY_ID }} --delete --only-show-errors dist/ ${{ secrets.TESTNET_S3_BUCKET }}

**Review by jonathansmirnoff — CHANGES_REQUESTED** [review]

File: .github/workflows/testnet-deploy.yml (hunk 4 above)
Comment (jonathansmirnoff) [inline .github/workflows/testnet-deploy.yml:37]:
```suggestion
          aws s3 sync --sse aws:kms --sse-kms-key-id ${{ secrets.TESTNET_KMS_KEY_ID }} --delete --only-show-errors dist/ ${{ secrets.TESTNET_S3_BUCKET }}
```

**Review by jonathansmirnoff — CHANGES_REQUESTED** [review]

File: .github/workflows/testnet-deploy.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+          node-version: "16"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build site
+        run: npm run build-with-tracking:testnet
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```
Comment (lucasvuotto) [inline .github/workflows/testnet-deploy.yml:30]:
```suggestion
        uses: aws-actions/configure-aws-credentials@v4
```

**Review by lucasvuotto — APPROVED** [review]
LGTM.

File: .github/workflows/testnet-deploy.yml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.TESTNET_ROLE_ARN }}
+          aws-region: ${{ secrets.TESTNET_AWS_REGION }}
+
+      - name: Deploy site to S3
+        run: |
+          aws s3 sync --sse aws:kms --sse-kms-key-id ${{ secrets.TESTNET_KMS_KEY_ID }} --delete --only-show-errors dist/ ${{ secrets.TESTNET_S3_BUCKET }}
```
Comment (lucasvuotto) [inline .github/workflows/testnet-deploy.yml:37]:
```suggestion
          aws s3 sync --sse aws:kms --sse-kms-key-id ${{ secrets.TESTNET_KMS_KEY_ID }} --delete --only-show-errors --exclude .gitkeep dist/ ${{ secrets.TESTNET_S3_BUCKET }}
```

**Review by ezequiel-rodriguez — APPROVED** [review]
LGTM

**Pull request merged**